src/App.js

```diff
@@ -1,39 +1,80 @@
 //TODO: STEP 1 - Import the useState hook.
+import { useState } from "react";
 import React from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  const [home, setHome] = useState(0);
+  const [away, setAway] = useState(0);
+
+
+  const handleReset = e => {
+    e.preventDefault();
+    setHome(0);
+    setAway(0);
+  };
 
   return (
     <div className="container">
       <section className="scoreboard">
         <div className="topRow">
           <div className="home">
-            <h2 className="home__name">Lions</h2>
+            <h2 className="home__name">Cowboys</h2>
 
             {/* TODO STEP 3 - We need to change the hardcoded values in these divs to accept dynamic values from our state. */}
 
-            <div className="home__score">32</div>
+            <div className="home__score">{home}</div>
           </div>
           <div className="timer">00:03</div>
```

Comment:
Juan you did a great job even adding the reset was a nice touch here. I would suggest trying your hand a the timmer when you get a chance. Here is a small article with moment.js https://upmostly.com/tutorials/build-a-react-timer-component-using-hooks